docs/src/explanation/troubleshooting.md

```diff
@@ -0,0 +1,26 @@
+# Troubleshooting
+
+This page provides techniques for troubleshooting common Canonical Kubernetes
+issues.
+
+## Invalid Kubeconfig
+
+### Problem
+
+The Kubernetes config file generated by the `k8s` CLI can not be used to access
+the cluster from an external machine.
+
+### Explanation
+
+A common technique for viewing a cluster Kubeconfig is by using the
+`kubectl config view` command.
+
+The `k8s kubectl` command invokes an integrated `kubectl` client. Thus
+`k8s kubectl config view` will output a seemingly valid Kubeconfig file.
+However, this will only be valid on cluster nodes where control plane services
+are available on localhost endpoints.
+
+### Solution
+
+Use `k8s config` instead of `k8s kubectl config` to generate a Kubeconfig file
+that is valid for use on external machines.
```

Comment:
Would this be the right spot to add example code on how to append the output of `k8s config` to an existing kubeconfig file?

Reply:
@louiseschmidtgen i don't think that fits well here -- at this point, you've already merged/appended the kubeconfig(s) and are dealing with the failure.

i think a separate page for tips and tricks / best practices would be a good place for managing external kubeconfigs, though that's out of scope for here.